test/WebJobs.Script.Tests/TestScripts/Node/NotificationHubOut/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+  "bindings": [
+    {
+      "type": "manualTrigger",
+      "direction": "in"
+    },
+    {
+      "type": "notficationhub",
+      "name": "notification",
+      "direction": "out",
+      "tagexpression": "foo||bar"
```

Comment:
What does this do? Allow NH subscription filtering on tags? Do you support binding parameters on this - e.g. a queue message comes in that has one or more tags defined in it, which you can auto bind?